opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -16,114 +16,157 @@
 from collections import OrderedDict
 from logging import getLogger
 from math import inf
+from threading import Lock
+from typing import Optional, Sequence, TypeVar
+from enum import IntEnum
 
-from opentelemetry._metrics.instrument import _Monotonic
+from opentelemetry.sdk._metrics.measurement import Measurement
+from opentelemetry.sdk._metrics.point import Gauge, Histogram, PointT, Sum
 from opentelemetry.util._time import _time_ns
 
+
+class AggregationTemporality(IntEnum):
+    AGGREGATION_TEMPORALITY_UNSPECIFIED = 0
+    AGGREGATION_TEMPORALITY_DELTA = 1
+    AGGREGATION_TEMPORALITY_CUMULATIVE = 2
```

Comment:
nit remove the `AGGREGATION_TEMPORALITY` prefix for ease of use and consistency with tracing https://github.com/open-telemetry/opentelemetry-python/blob/dfb5c66ae310001bb40326f6286345b7fa252aba/opentelemetry-api/src/opentelemetry/trace/__init__.py#L157-L170

Reply:
All right, removed